.github/workflows/sycl-cts-linux-cpu-latest-nightly.yml

```diff
@@ -0,0 +1,20 @@
+name: SYCL-CTS on Linux&CPU with the latest nightly
+run-name: SYCL-CTS - ${{ github.ref_name }}
+
+on:
+  push:
+    tags:
+      - nightly-*
```

Comment:
Can we add manual workflow_dispatch as well? Possibly with a selectable runner/target devices.

Reply:
As discussed offline we can just run `sycl-linux-run-tests.yml` as it has `workflow_dispatch`.